tests/unit/core/server/request/test-class-core-get-charge-request.php

```diff
@@ -43,12 +43,19 @@ public function set_up() {
 	public function test_exception_will_throw_if_charge_id_is_not_set() {
 		$this->expectException( Invalid_Request_Parameter_Exception::class );
 
-		$request = new Get_Charge( $this->mock_api_client, $this->mock_wc_payments_http_client, null );
+		new Get_Charge( $this->mock_api_client, $this->mock_wc_payments_http_client, null );
 	}
+
 	public function test_exception_will_throw_if_charge_id_is_invalid() {
 		$this->expectException( Invalid_Request_Parameter_Exception::class );
-		$request = new Get_Charge( $this->mock_api_client, $this->mock_wc_payments_http_client, '1' );
+		new Get_Charge( $this->mock_api_client, $this->mock_wc_payments_http_client, '1' );
+	}
+
+	public function test_py_prefix_will_not_throw_exception() {
+		new Get_Charge( $this->mock_api_client, $this->mock_wc_payments_http_client, 'py_xyz' );
+		$this->addToAssertionCount( 1 ); // We're not asserting anything, just not expecting an exception.
```

Comment:
Another way to achieve this would be to use `$this->expectNotToPerformAssertions();` I think, but the two approaches don't differ from each other almost at all, so I'm leaving this comment just as an easy thought which doesn't require any changes! 👍